src/components/Picture/stories.tsx

```diff
@@ -19,6 +19,24 @@ const params = { assetId: process.env.SANITY_ASSET_ID };
 export default {
     title: 'Picture',
     component: Picture,
+    args: {
+        builderOptions: {
+            blur: undefined,
+            sharpen: undefined,
+            format: undefined,
+            invert: undefined,
+            orientation: undefined,
+            quality: undefined,
+            fit: undefined,
+            crop: undefined,
+            saturation: undefined,
+            auto: undefined,
+        },
+        media:[{
```

Comment:
Not sure if this `media` option should be supplied by default, might make sense to add a second story specifying this, to show how it works.